Azure/communication-ui-library-ios
# [Feature] Call Diagnostics feature implemented

## Purpose
New CSS feature to allow users see the call diagnostics. 
New API for developers to fetch the last known call id

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test

- Join a call
- In call view notice the new overflow menu to get the call diagnostics information.
